### ISSUE TITLE
Pylance doesn't care "positional only parameters"

### ISSUE THREAD

**Issue by PASUNX** [microsoft/pylance-release#1109]
<h3> Environment data </h3>

-   Language Server version: 2021.3.4
-   OS and version: win32 x64
-   Python version (and distribution if applicable, e.g. Anaconda): 
-   python.analysis.indexing: undefined
-   python.analysis.typeCheckingMode: strict

<h3> Code Snippet / Additional information </h3>

```python
def func(name: str, /, **kwargs: str) -> None:
    print(name, kwargs)

func("PASUNX", name = "PASUNX") # error:
                                # Parameter "name" is already assigned Pylance (reportGeneralTypeIssues)
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.4.0, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202140-7-april-2021
